### PR TITLE
Feature/webdriver manager

### PR DESCRIPTION
**Pull request by SergeyPirogov**

### CODE REVIEW

File: src/main/java/com/codeborne/selenide/webdriver/WebDriverBinaryManager.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            } else if (isMarionette() &&
+                    isEmtyProperty(MARIONETTE_PROPERTY)) {
+                FirefoxDriverManager.getInstance().setup();
+            }
+        } catch (final Exception ex) {
+            log.warning("Problem to load driver binary for " + Configuration.browser);
+        }
+    }
+
+    private static boolean isEmtyProperty(String property){
```
Comment (ddemin) [inline src/main/java/com/codeborne/selenide/webdriver/WebDriverBinaryManager.java]: Typo, fix please )

File: src/test/java/integration/AutomaticDriverManagementTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Test
+    public void canStartChromeWithAutomaticDriver() throws Exception {
+        Configuration.browser = WebDriverRunner.CHROME;
+        Selenide.open("http://automation-remarks.com");
+        Selenide.$$(".post").shouldHave(CollectionCondition.size(9));
+    }
+
+    @Test
+    public void canStartPhantomJSWithAutomaticDriver() throws Exception {
```
Comment (ddemin) [inline src/test/java/integration/AutomaticDriverManagementTest.java]: What is the purpose of this test?

**Comment by SergeyPirogov** [comment]
@ddemin everything is fixed

**Comment by asolntsev** [comment]
@SergeyPirogov Some tests are failing on Travis CI. 
Please investigate, this is test reports: https://transfer.sh/c2qD6/1186.1-log.tar.gz

**Comment by SergeyPirogov** [comment]
@asolntsev The fail is in test **integration.ConfirmTest**

```
org.openqa.selenium.WebDriverException: unknown error: unhandled inspector error: {"code":-32603,"message":"Could not handle JavaScript dialog"}
```

Seems that ist's known problew with chrome and allerts

**Comment by asolntsev** [comment]
@SergeyPirogov Class `AutomaticDriverManagementTest` changes the global variable `WebDriverRunner.browser`. The following tests will be executed in this browser (instead of configured one). You should either restore browser after your tests or use `assumeTrue(isChrome())` / `assumeTrue(isMarionette())` inside test.

**Comment by thasherwin** [comment]
What does this closing means. That the pull request was not successful?

**Comment by SergeyPirogov** [comment]
There is some problem with **driver_manager** and **Travis CI**. Driver manager downloads driver zip, but than says that driver can not be reached. Need to dig deeper into this and then will create new request

**Comment by thasherwin** [comment]
Okay. Clear.

**Pull request closed without merging**